src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -118,6 +120,10 @@ public static class Constants
 
         public const string DotNetFramework46 = ".NETFramework,Version=v4.6";
 
+        public const string DotNetFrameworkCore10 = ".NETCoreApp,Version=v1.0";
```

Comment:
What about DotNetFrameworkCore20?

Reply:
We don't need it. For .net core, we interpret dynamically from the app's runtime json.